src/spryker/docker/image/console/root/lib/task/spryker/install.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+function task_spryker_install()
+{
+    passthru "PGPASSWORD=$DB_PASS vendor/bin/install -r docker -x frontend"
+    passthru "PGPASSWORD=$DB_PASS vendor/bin/install -r docker -s database-migrate"
+    passthru "PGPASSWORD=$DB_PASS vendor/bin/install -r docker -s demodata"
```

Comment:
do we always need the demo data? Any use beyond getting a base sample environment? Does it need to be removed when real data is provided?

Reply:
I'll make it optional, off by default.